test/test_utils.py

```diff
@@ -77,6 +78,23 @@ def test_incorrect(self):
                              "c:\\test\\te\"st2")
 
 
+class ExtractYearTest(PicardTestCase):
+    def test_string(self):
+        self.assertEqual(extract_year_from_date(""), None)
+        self.assertEqual(extract_year_from_date(2020), None)
+        self.assertEqual(extract_year_from_date("2020"), 2020)
+        self.assertEqual(extract_year_from_date('2020-02-28'), 2020)
+        self.assertEqual(extract_year_from_date('2015.02'), 2015)
+        self.assertEqual(extract_year_from_date('2015; 2015'), None)
```

Comment:
I'd add a test for the format as supported by ID3 [1]: `yyyy-MM-ddTHH:mm:ss`

```
self.assertEqual(extract_year_from_date('2020-07-21T13:00:00'), 2020)
```

[1] https://id3.org/id3v2.4.0-structure

Reply:
Done in 5e5535a